src/Castle.Services.Logging.SerilogIntegration/Castle.Services.Logging.SerilogIntegration.csproj

```diff
@@ -58,22 +58,12 @@
     <ErrorReport>prompt</ErrorReport>
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'NET40-Release|AnyCPU'">
```

Comment:
Could you please remove the .NET 4.0 and 3.5 sections completely, not sure why VS doesn't do that for you.